pkg/controller/networkpolicy/clusternetworkpolicy.go

```diff
@@ -253,12 +256,12 @@ func (n *NetworkPolicyController) deleteNamespace(old interface{}) {
 // instance to the caller wherein, it will be either stored as a new Object
 // in case of ADD event or modified and store the updated instance, in case
 // of an UPDATE event.
-func (n *NetworkPolicyController) processClusterNetworkPolicy(cnp *crdv1alpha1.ClusterNetworkPolicy) *antreatypes.NetworkPolicy {
```

Comment:
I think Yang had a comment about breaking down this method if possible?

Reply:
I separated `processRules` func. Do I misunderstand it?

Reply:
sorry, I missed that
LGTM